NodeOS.svg

```diff
@@ -3,6 +3,7 @@
    xmlns="http://www.w3.org/2000/svg"
    width="56cm"
    height="60cm"
+   shape-rendering="geometricPrecision"
```

Comment:
Can you show both to compare?

El 9/8/2016 16:35, "John Green" <notifications@github.com> escribió:

> In NodeOS.svg
> <https://github.com/NodeOS/media/pull/3#discussion_r74070287>:
>
> > @@ -3,6 +3,7 @@
> >     xmlns="http://www.w3.org/2000/svg"
> >     width="56cm"
> >     height="60cm"
> > +   shape-rendering="geometricPrecision"
>
> The improvements are somewhat of the same. CrispEdges produces a
> "color-true" version with a bit of "damage" to the borders while
> geometricPrecission produces a, well, geometrically precise image. It's up
> to you :P
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/NodeOS/media/pull/3/files/231d44d1dcfea0e3485b9048a4c589cd61cfc889#r74070287>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAgfvjQuYgggeq-P6WrRK6gM7aXX-L4-ks5qeJArgaJpZM4JgHOI>
> .
>